lumen/ai/coordinator.py

```diff
@@ -247,6 +247,7 @@ async def run_demo(event):
                     button_style="outline",
                     on_click=use_suggestion,
                     margin=5,
+                    disabled=state.param.busy
```

Comment:
This worries me a little, `state.param.busy` can rapidly toggle on and off which would result in UI flicker.

Reply:
These are only on the suggestion buttons that disappear after the first message; shouldn't affect much.

Reply:
Maybe we can implement [debouncing](https://www.geeksforgeeks.org/debouncing-in-javascript/) in param:

```
with button.param.update(disabled=True, debounce=2000)
```

Reply:
In general that might be nice but it's somewhat difficult. Also that API won't work since it'd clash with a parameter called `debounce`. For now I'd suggest a difference approach here.

Reply:
Ok I linked it to interface loading.